clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -680,63 +696,63 @@ export const TcfOverlay = () => {
           </ConsentBanner>
         );
       }}
-      renderModalContent={
-        !experienceFull
-          ? undefined
-          : () => (
-              <TcfTabs
-                experience={experienceFull}
-                customNotices={privacyNoticesWithBestTranslation}
-                enabledIds={draftIds}
-                onChange={handleToggleChange}
-                activeTabIndex={activeTabIndex}
-                onTabChange={handleTabChange}
+      renderModalContent={() => (
+        <TcfTabs
+          experience={experienceFull || experienceMinimal}
+          customNotices={privacyNoticesWithBestTranslation}
+          enabledIds={draftIds}
+          onChange={handleToggleChange}
+          activeTabIndex={activeTabIndex}
+          onTabChange={handleTabChange}
+          fullExperienceState={fullExperienceState}
+        />
+      )}
+      renderModalFooter={({ onClose }) => {
+        const onSave = (consentMethod: ConsentMethod, keys: EnabledIds) => {
+          handleUpdateAllPreferences(consentMethod, keys);
+          onClose();
+        };
+        return (
+          <TcfConsentButtons
+            experience={experienceFull || experienceMinimal}
+            onAcceptAll={() => {
+              handleAcceptAll();
+              onClose();
+            }}
+            onRejectAll={() => {
+              handleRejectAll();
+              onClose();
+            }}
+            renderFirstButton={() => (
+              <Button
```

Comment:
<img width="1394" height="403" alt="image" src="https://github.com/user-attachments/assets/ec74e449-568e-48b4-b250-f8c594295d38" />

Should the button outline still be black? IMO this makes the button look "focused". Is there a gray we can use here instead?

Reply:
This is the "Primary" color, and in our demo's case the primary color is set to our Minos color. It won't necessarily be this color for everyone. (Cookie house is some kind of red currently)

Reply:
Oh, sorry. I'm trying to move to fast. I missed the word "outline" and thought you were talking about the black buttons... Yeah, I like your suggestion. I'll update the disabled styling.